README.md

```diff
@@ -118,6 +118,30 @@ h.RegisterCheck(&health.Config{
 })
 ```
 
+#### Ping built-in check(s)
+The ping checks allow you to verifies that a resource is still alive and reachable.
+For example, you can use it as a DB ping check (`sql.DB` implements the Pinger interface):
+```go
+	db, err := sql.Open(...)
+	dbCheck, err := checks.NewPingCheck("db.check", db, time.Millisecond*100)
```

Comment:
I would probably add a `checks.Must` function here with the `func(check Check, err error)` signature, so that checks initialized in a `main` function can panic. I find a commonly applied practice to interrupt application bootstrap and exit with panic if something goes wrong.

Obviously that's optional and could be part of a subsequent PR.

Reply:
Makes sense, but I need to read about this pattern. I'm still green in golang ;)